### PR TITLE
(Cypress) Add delegator DB backup

### PR DESCRIPTION
**Pull request by jay-kim-kor**

### CODE REVIEW

File: cronjobs/values.yaml [hunk 1]
```diff
@@ -27,18 +27,28 @@ jobs:
...
         echo "Step 3: Upload all csv files to GCS bucket."
         gcloud storage cp /data/csv/*.csv gs://orakl-db-backup-cypress
 
         echo "Step 4: Delete old data tables"
-        psql $DATABASE_URL -t -c "SELECT tablename FROM pg_tables WHERE tablename LIKE 'data_%';" \
-        | xargs -I {} psql $DATABASE_URL -c "DROP TABLE ${};"
+        psql $ORAKL_DB_URL -t -c "SELECT tablename FROM pg_tables WHERE tablename LIKE 'data_%';" \
+        | xargs -I {} psql $ORAKL_DB_URL -c "DROP TABLE {};"
+        psql $DELEGATOR_DB_URL -t -c "SELECT tablename FROM pg_tables WHERE tablename LIKE 'transactions_%';" \
+        | xargs -I {} psql $DELEGATOR_DB_URL -c "DROP TABLE {};"
```
Comment (martinkersner) [inline cronjobs/values.yaml:42]: @jay-kim-kor Shouldn't we delete the CSV files after we uploaded them to bucket?

Comment (jay-kim-kor) [reply cronjobs/values.yaml:42]: Step 3 is uploading csv files part. Also this part doesn't delete csv files. It'd delete archived table after upload csv files.
The csv files will deleted with pod when job was completed or failed.

Comment (martinkersner) [reply cronjobs/values.yaml:42]: Got it! Thank you!

**Review by martinkersner — APPROVED** [review]
LGTM!

**Pull request merged**